home/ivp-planning/user-interface/the-2d-panel.md

```diff
@@ -1,33 +1,29 @@
-| ![](../../../.gitbook/assets/iVP\_icon\_reference\_point.jpg)             | Sets the virtual zero / the reference point of the [grid](the-grid.md) and the [coordinate system](the-grid.md)                                                                                                                                                                                              |
-| ![](../../../.gitbook/assets/iVP\_icon\_grid\_on\_off.jpg)                | [Toggles the visibility of the grid](the-grid.md#displaying-and-hiding-the-grid)                                                                                                                                                                                                                             |
-| ![](../../../.gitbook/assets/iVP\_icon\_animations\_on\_off.jpg)          | Toggles all [animations ](../machines/animations.md)globally without overwriting the animation values set for                                                                                                                                                                                                |
-| ![](../../../.gitbook/assets/iVP\_icon\_imported\_pdf\_on\_off.jpg)       | Toggles the visibility of the [imported PDF floor plan](../getting-started/importing-pdfs.md)                                                                                                                                                                                                                |
-
-## Navigation in the the 2D-Panel:
-
-To change the visible area, you can move the camera by holding down the **middle mouse button**. The zoom level can be adjusted by using the **mouse wheel**.
+|                                                                            |                                                                                                                                                            |
+| -------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------  |
+| ![](../../../.gitbook/assets/2D_panel/Icon_regular_mode.jpg)               | Switches to a mode that lets you drag and drop machines and objects                                                                                        |
+| ![](../../../.gitbook/assets/2D_panel/Icon_path_mode.jpg)                  | Switches to the [path tool mode](../advanced-tools/path-tool.md) which allows you to draw [measurements](../advanced-tools/path-tool.md#measurements), [fences](../advanced-tools/fence-tool.md) and [rooms](../advanced-tools/the-room-tool.md)                                                                                                                                                                                       |
```

Comment:
The header does not exist in ../advanced-tools/path-tool.md#measurements. This has been like this before, but we could fix it in one go.